kitsune/sumo/static/sumo/js/browserdetect.js

```diff
@@ -130,6 +136,11 @@ var BrowserDetect = {
       identity: 'win'
     },
     {
+      string: navigator.userAgent,
+      subStrings: [/iPad|iPhone/],
```

Comment:
Huh, I guess, but only by accident. I wrote this line thinking "I'll add the test for iPod touch, and see how it fails", and then forgot when it didn't fail. I'll add it.